drizzle/0021_green_maestro.sql

```
@@ -0,0 +1,19 @@
+	"storage_path" text NOT NULL,
+	"uploaded_at" timestamp with time zone DEFAULT now() NOT NULL,
+	"created_at" timestamp with time zone DEFAULT now() NOT NULL,
+	"updated_at" timestamp with time zone DEFAULT now() NOT NULL
+);
+--> statement-breakpoint
+ALTER TABLE "documents" ENABLE ROW LEVEL SECURITY;--> statement-breakpoint
+ALTER TABLE "documents" ADD CONSTRAINT "documents_account_id_accounts_id_fk" FOREIGN KEY ("account_id") REFERENCES "public"."accounts"("id") ON DELETE no action ON UPDATE no action;--> statement-breakpoint
+CREATE INDEX "documents_account_id_created_at_idx" ON "documents" USING btree ("account_id","created_at" DESC NULLS LAST);--> statement-breakpoint
+CREATE UNIQUE INDEX "documents_account_id_storage_path_unique_idx" ON "documents" USING btree ("account_id","storage_path");--> statement-breakpoint
+GRANT SELECT, INSERT ON TABLE "documents" TO authenticated;--> statement-breakpoint
+CREATE POLICY "documents_owner_select" ON "documents" FOR SELECT TO authenticated USING ("account_id" = (SELECT "id" FROM "accounts" WHERE "auth_user_id" = "app"."current_auth_subject"()));--> statement-breakpoint
```

SQLFluff (4.1.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)